scripts/generate_key.sh

```diff
@@ -0,0 +1,205 @@
+#!/bin/bash
+set -e
+
+KEY_FORMAT=$1 # APT, RPM or TARGZ
+
+if [ -z $KEY_FORMAT ];then
+  echo -e "${RED} usage ./generate_key.sh [APT|RPM|TAGGZ] .${NC}"
+      exit 1
+    fi
+
+# params: KEY_FORMAT
+PASSPHRASE_LENGTH=24
+KEY_LIFE_TIME="10y"
+EMAIL="caos-team@newrelic.com"
+OUTPUT_DIR="$HOME/caos_key_generator"
+LOG_FILE="$OUTPUT_DIR/key_generation.log"
+RED='\033[0;31m'
+NC='\033[0m' # No Color
+
+#check dependencies
+DEP="gpg openssl docker"
+for dep in $DEP;do
+  if [ ! $( which "$dep" ) ];then
+    echo -e "${RED}$dep is mandatory. Install it and restart the process.${NC}"
+    exit 1
+  fi
+done
+
+echo "Existing keys sanity check..."
+#sanity check
+KEYS_COUNT=$(gpg --list-secret-keys --keyid-format=long 2>$LOG_FILE | grep "${EMAIL}" | wc -l | grep -oE "[0-9]*")
+if [ "${KEYS_COUNT}" != "0" ]
+then
```

Comment:
We have `then` in the same line in one if above. I think we should pick a style and be consistent with it.